Adds Discover help center article (#15)
elastic/docs-content: help-center/discover.mdx

This article shows you how to use Discover to understand your data better. You
will learn how to:

* select data views
* explore your fields
* define runtime fields
* search and filter your data
* use ES|QL
* save search
* visualize field values

You can also take a guided tour about Discover on the UI by clicking
**Take the tour** under the **Get the best look at your search results**
section.

## Select or create a data view

Select the data you want to explore. You need to create a data view from your
Elasticsearch data to access it through Discover.

<DocCallOut color="success" title="Data view">
A data view can point to one or more indices, data streams, or index aliases to
represent your Elasticsearch data based on the requirements you define. For
example, a data view can point to your log data from yesterday, or all indices
that contain your data.
</DocCallOut>

Adjust the time range to fit your requirements. If your data is not
time-based, range selection is not available.

To view the count of documents for a given time in the specified range,
select the corresponding part of the chart.

## Explore fields

> Discover includes a table that shows all the documents that match your search.

Author: Good point, I rephrased the first sentence.

include or exclude specific fields, filter for a value in a range, and more.

Click <DocIcon type="plusInCircleFilled" title="Add filter" /> next to the query
bar to configure a filter.

## Enable ES|QL

[The Elasticsearch Query Language (ES|QL)](https://www.elastic.co/guide/en/elasticsearch/reference/current/esql.html)
provides a powerful way to filter, transform, and analyze data stored in
Elasticsearch.

You can use ES|QL in **Discover** to query and aggregate your data, create
visualizations, or set up alerts. To swtich to ES|QL mode, click the Data views
selector and select **Language: ES|QL**.

Use the query bar to create ES|QL queries. Refer to the
[documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/esql-kibana.html)
to learn more about ES|QL in **Discover**.

## Save search

Click ***Save** in the toolbar to save your search configuration and the search
results. You can use it later to generate a CSV report, create visualizations or
dashboards. Saving a search saves the query text, filters, and the current view
of **Discover**, including the columns selected in the document table, the sort
order, and the data view.

## Visualize field values

> If the values of a field can be aggregated, you can visualize it from Discover.

Reviewer: ```suggestion
When the values of a field can be aggregated, you can visualize the field in Discover.
```
